distribution/lib/Standard/Database/0.0.0-dev/src/Internal/JDBC_Connection.enso

```diff
@@ -117,6 +117,26 @@ type JDBC_Connection
                 compiled = query.prepare
                 go compiled.first compiled.second
 
+    ## PRIVATE
```

Comment:
I think this should have a comment explaining why it was necessary for SQLServer